src/Plugin/Validators/ValueInList.php

```diff
@@ -28,6 +29,47 @@ class ValueInList extends TripalCultivateValidatorBase {
   use ColumnIndices;
   use ValidValues;
 
+  /**
+   * A mapping of all of the tokens supported by this validator.
+   *
+   * @var array
+   *   An associative array mapping tokens to their details, such as the
+   *   developer case string and the default message to substitute the token.
+   *   The following tokens are implemented for this mapping, with the following
+   *   descriptions for their 'default-msg' values:
+   *   - 'table-invalid': the token containing the message used by cases where
+   *     data in row and column combination is invalid value or failed
+   *     a case insensitive match.
+   *   Tokens below cannot be overriden as their value is determined at runtime:
```

Comment:
This will make it consistent with ValidDelimitedFile:
```suggestion
   *   Tokens below cannot be overriden as their value is determined at runtime
   *   or case messages are designed to cover multiple cases. See above for
   *   tokens that can be changed.
```